scripts/start

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+set -e
```

Comment:
the following if/fi came from existing code and likely is redundant with the new linting action